README.md

```diff
@@ -1163,10 +1163,12 @@ Table of Contents
   * [AllTheFreeStock](https://allthefreestock.com) - a curated list of free stock images, audio and videos.
   * [Ant Design Landing Page](https://landing.ant.design/) - Ant Design Landing Page provides a template built by Ant Motion's motion components. It has a rich homepage template, downloads the template code package, and can be used quickly. You can also use the editor to quickly build your own dedicated page.
   * [Backlight](https://backlight.dev/) — With collaboration between developers and designers at heart, Backlight is a very complete coding platform where teams build, document, publish, scale and maintain Design Systems.Free plan allows up to 3 editors working on 1 design system with unlimited viewers.
+  * [Blob Generator](https://www.magicpattern.design/tools/blob-generator) — Create random organic-shaped blobs with solid colors, linear/radial gradients, and pattern fills.
   * [BoxySVG](https://boxy-svg.com/app) — A free installable Web app for drawing SVGs and exporting in svg,png,jpeg an other formats.
   * [clevebrush.com](https://www.cleverbrush.com/) — Free Graphics Design / Photo Collage App, also they offer paid integration of it as component.
   * [cloudconvert.com](https://cloudconvert.com/) — Convert anything to anything. 208 supported formats including videos to gif.
   * [CodeMyUI](https://codemyui.com) - Handpicked collection of Web Design & UI Inspiration with Code Snippets.
+  * [CSS Gradient Generator](https://www.magicpattern.design/tools/gradient-generator) — Create, save & share beautiful CSS linear, radial, and conic gradients for your backgrounds.
```

Comment:
I'd rather we just list https://www.magicpattern.design/ and say it has a bunch of things.

Reply:
Of course, no worries. If you need a description for the service, you can use sth like this: 

"A collection of SVG & CSS generators for gradients, blobs, patterns, and more."

And you link to the page with all the tools directly: [https://www.magicpattern.design/tools](https://www.magicpattern.design/tools). 

Thanks in advance!

Reply:
Yes, please update the PR as you suggest and we can merge it.

Reply:
Awesone. Done! 

Let me know if everything looks fine